src/core/continua_dynamiclink.tf

```diff
@@ -76,17 +75,18 @@ resource "azurerm_service_plan" "continua" {
 }
 
 module "appservice_continua" {
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//app_service?ref=v7.28.0"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//app_service?ref=v7.33.0"
 
   name                = format("%s-app-continua", local.project)
   resource_group_name = azurerm_resource_group.continua_rg.name
   location            = azurerm_resource_group.continua_rg.location
 
-  app_command_line  = "yarn start"
-  health_check_path = "/health"
-  node_version      = "18-lts"
-  app_settings      = local.continua_appsvc_settings
-  sticky_settings   = []
+  app_command_line             = "yarn start"
+  health_check_path            = "/health"
+  health_check_maxpingfailures = 3
+  node_version                 = "18-lts"
+  app_settings                 = local.continua_appsvc_settings
+  sticky_settings              = []
```

Comment:
some functions/app service needs sticky_settings (deployment slot in Azure UX).
This app service it's ok but probably we need to check other function/app service